internal/app/singularity/oci_create_linux.go

```diff
@@ -22,7 +21,7 @@ import (
 
 // OciCreate creates a container from an OCI bundle
 func OciCreate(containerID string, args *OciArgs) error {
-	starter := buildcfg.LIBEXECDIR + "/singularity/bin/starter"
+	starterBinary, _ := exec.LookStarterPath(false, false)
```

Comment:
For readability's sake, you should add a function in that exec package that I'm guessing should be called `GetStarterPathNoSuid()`